builder/bundle_control.go

```diff
@@ -769,19 +769,22 @@ func createBundleFile(bundle string, path string) error {
 // CreateBundles copies a list of bundles from upstream-bundles to local-bundles
 // if they are not already there or creates a blank template if they are new.
 // 'add' will also add the bundles to the mix.
-func (b *Builder) CreateBundles(bundles []string, add bool, git bool) error {
+func (b *Builder) CreateBundles(bundles []string, add bool, git bool, local bool) error {
 	// Fetch upstream bundle files if needed
 	if err := b.getUpstreamBundles(); err != nil {
 		return err
 	}
 	var err error
 	for _, bundle := range bundles {
 		path, _ := b.getBundlePath(bundle)
+		localPath := filepath.Join(b.Config.Mixer.LocalBundleDir, bundle)
+		if _, err = os.Stat(localPath); err == nil {
+			fmt.Printf("Bundle %q already exists in %q; skipping\n", bundle, localPath)
```

Comment:
I like that you used `localpath` instead of just `b.Config.Mixer.LocalBundleDir` so the user can directly copy and edit.
Maybe just reword it to `fmt.Printf("Bundle %q already exists at %q; skipping\n", bundle, localPath)`

I might have another PR to keep the bundle command messages consistent.